packages/shared/lib/transactionHistory.ts

```diff
@@ -70,7 +70,8 @@ export const generateTransactionHistoryCsvFromAccount = (
 }
 
 export const generateTransactionHistoryFileName = (profileName: string, accountAlias: string): string => {
-    const DEFAULT_FILE_NAME_PATTERN = 'firefly-transaction-history-{{profileName}}-{{accountAlias}}-{{date}}'
+    const DEFAULT_FILE_NAME_PATTERN =
+        'iota-legacy-migration-tool-transaction-history-{{profileName}}-{{accountAlias}}-{{date}}'
```

Comment:
It's going to be a very long name, right?
I think it would be a good idea to think of a shorter one, wdyt?